messaging/methods/messages/createMessage.md

```diff
@@ -51,6 +52,7 @@ When sending a group message to an invalid phone number, you may receive extrane
 | tag           | `string` | An custom String that you can use to track this particular message                                                                                                                                                                                                                                                                                                  |
 | segmentCount  | `int`    | This indicates the number of segments the original message from the user is broken into before sending over to carrier networks                                                                                                                                                                                                                                      |
 | priority      | `string` | The priority specified by the user |
+| expiration    | `string` | Message expiration date and time specified by the user. Note that this field only populates in the response body if it was specified in the corresponding create message request.
```

Comment:
IMO this second sentence isn't necessary since it's true for all optional fields (tags, media, priority, etc). Plus it's implied by the first sentence anyway.